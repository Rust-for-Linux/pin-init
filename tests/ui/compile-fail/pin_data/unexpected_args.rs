use pinned_init::*;

#[pin_data(Bar)]
struct Foo {}

fn main() {}
