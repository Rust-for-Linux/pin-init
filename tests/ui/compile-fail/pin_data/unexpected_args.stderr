error: compile_error! takes 1 argument
 --> tests/ui/compile-fail/pin_data/unexpected_args.rs:3:1
  |
3 | #[pin_data(Bar)]
  | ^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `$crate::__pin_data` which comes from the expansion of the attribute macro `pin_data` (in Nightly builds, run with -Z macro-backtrace for more info)

error: Wrong parameters to `#[pin_data]`, expected nothing or `PinnedDrop`, got '{}'.
 --> tests/ui/compile-fail/pin_data/unexpected_args.rs:3:1
  |
3 | #[pin_data(Bar)]
  | ^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `$crate::__pin_data` which comes from the expansion of the attribute macro `pin_data` (in Nightly builds, run with -Z macro-backtrace for more info)
